discord/interactions.py

```diff
@@ -306,12 +318,12 @@ def is_component(self) -> bool:
         return self.type == InteractionType.component
 
     @utils.cached_slot_property("_cs_channel")
+    @utils.deprecated("Interaction.channel", "2.7", stacklevel=4)
```

Comment:
Why deprecate

Reply:
Nelo said I should deprecate instead of remove since I removed it in the beginning.
It makes no sense to have it imo, it provides at best the same as `.channel` and at worst just partials that serve no purpose.